controllers/users.js

```diff
@@ -415,11 +459,16 @@ class Users {
         });
       }
 
-      const user = await User.findOne({ where: { id }, attributes: ['following'] });
```

Comment:
if a line is not too long, you should keep it as it is in order to avoid have a lot of lines of code, you are doing that almost every where